subprojects/docs/src/docs/userguide/compatibility.adoc

```diff
@@ -20,8 +20,8 @@ The sections below describe Gradle's compatibility with several integrations.
 Versions not listed here may or may not work.
 
 == Java
-A Java version between 8 and 19 is required to execute Gradle.
-Java 20 and later versions are not yet supported.
+A Java version between 8 and 20 is required to execute Gradle.
+Java 21 and later versions are not yet supported.
```

Comment:
Apologies for jumping in abruptly; I noticed that Gradle recently merged a PR that upgrades ASM to 9.5 which brings forth support for Java 21. Does this still mean that Java 21 cannot be supported by the likes of Gradle 8.3?

Reply:
No worry :-)

We plan to certify Java 21 support through toolchains for Gradle 8.4. You might try with 8.3, but we cannot guarantee anything.